### PR TITLE
Fix version edit url for ajax toggle operation

### PR DESCRIPTION
**Pull request by rabauss**
Unfortunately the version edit url is not correct while using ajax toggle.
I was not sure if other operations are handled as well with the AjaxOperations, because of that I added the check `$operation !== 'toggle'` - maybe that is not neccesary...

### CODE REVIEW

File: library/Haste/Dca/AjaxOperations.php [hunk 1]
```diff
@@ -303,4 +305,16 @@ private function getOptions(array $hasteAjaxOperationSettings)
     {
         return (array) $hasteAjaxOperationSettings['options'];
     }
+
+    private function getVersionEditUrl(int $id, string $operation)
```
Comment (qzminski) [inline library/Haste/Dca/AjaxOperations.php:309]: Haste can be still used by PHP 5.6 systems so please don't use strict types here.

**Comment by Toflar** [comment]
Should be no issue because we're checking for `hasteAjaxOperation`.

**Comment by Toflar** [comment]
Otherwise that looks good imho, thanks :)

File: library/Haste/Dca/AjaxOperations.php (hunk 1 above)
Comment (rabauss) [reply library/Haste/Dca/AjaxOperations.php]: It's really difficult not to use this new stuff anymore

**Comment by qzminski** [comment]
I have just released 4.24.7, thank you @rabauss !

**Pull request merged**